mvebu64boot.c

```diff
@@ -701,6 +701,8 @@ static bool loop_boot_pattern(int fd)
 			fprintf(stderr, "Error: Failed to read from tty device: %s\n", strerror(errno));
 			break;
 		}
+		fputc(byte, stderr);
+		fflush(stderr);
```

Comment:
What is this hack? Garbage from console must **not** be printed, neither to stdout nor to stderr.

Reply:
So then problem is somewhere else...